packages/acs-ui-common/src/identifier.ts

```diff
@@ -24,8 +24,7 @@ export const toFlatCommunicationIdentifier = (identifier: CommunicationIdentifie
  */
 export const fromFlatCommunicationIdentifier = (id: string): CommunicationIdentifier => {
   // if the id passed is a phone number we need to build the rawId to pass in
-  const rawId = id.indexOf('+') === 0 ? '4:' + id.slice(1) : id;
-  return createIdentifierFromRawId(rawId);
+  return createIdentifierFromRawId('4:' + id);
```

Comment:
do we know that this is a valid raw ID? I thought for it to be the raw one it needed to be `4:xxxxxxxxxx`?

Reply:
I think I had to fix up some of the logic but the big change now is that they are not going to change the "id" so now we need to include the + as well.